snap/snapcraft.yaml

```
@@ -343,7 +343,10 @@ parts:
     # since it changes rarely and therefore will not trigger a new pull
     source: snap/local/build-helpers
     override-pull: |
-      snapcraftctl set-version $(cat $SNAPCRAFT_PROJECT_DIR/VERSION)-$(date +%Y%m%d)+$(git rev-parse --short HEAD)
+      cd $SNAPCRAFT_PROJECT_DIR
+      PROJECT_VERSION=$(cat VERSION)
+      GIT_REVISION=$(git rev-parse --short HEAD)
+      snapcraftctl set-version ${PROJECT_VERSION}-$(date +%Y%m%d)+${GIT_REVISION}}
```

too many `}`s here, try:

```suggestion
      snapcraftctl set-version ${PROJECT_VERSION}-$(date +%Y%m%d)+${GIT_REVISION}
```